geowrangler/grids.py

```diff
@@ -231,10 +231,25 @@ def generate_grid(
         vertices, boundary, northing_col="y", easting_col="x"
     )
 
-    tiles_in_geom = polygon_fill.fast_polygon_fill(vertices, unique_id_col)
-
+    polygon_fill_result = polygon_fill.fast_polygon_fill(vertices, unique_id_col)
+    tiles_in_geom = polygon_fill_result["tiles_in_geom"]
     bboxes = self._xy_to_bbox(tiles_in_geom, boundary, "x", "y")
 
+    # this is error correction on the polygon boundary (not the square boundary)
```

Comment:
ah yes there's a section in `15_polygon_fill.ipynb`!

> Why can't we just add all of the off-boundary pixels? We can't because not all of the off-boundary pixels actually intersect the polygon when in geographic coordinates.